### PR TITLE
Fixes #39 Reformatting the code according to PEP style guide.

### LINKED ISSUE #39: Reformatting the code according to PEP style guide.
The code in check _addon.py, check_repo.py and common.py needs to be reformatted according to PEP 8 style guide along with some improvements in code like removing unnecessary parenthesis.
Please let me know if this sounds reasonable and I can start working on a pull for this!
Regards
Aditya

### PR DESCRIPTION
**Pull request by AdityaGupta030697**
Fixes #39
made changes according to PEP style guide.

### CODE REVIEW

File: check_addon.py [hunk 1]
```diff
@@ -334,11 +338,11 @@ def _check_file_whitelist(error_counter, file_index, addon_path):
 
 def relative_path(file_path):
     path_to_print = file_path[len(REL_PATH):]
-    return ("." + path_to_print)
+    return "." + path_to_print
```
Comment (enen92) [inline check_addon.py]: ".{}".format()

Comment (AdityaGupta030697) [reply check_addon.py]: Done. thanks for the review.

File: .idea/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Razzeee) [inline .idea/vcs.xml]: This file should not be here. Please remove it and maybe add `.idea` to our gitignore

Comment (AdityaGupta030697) [reply .idea/vcs.xml]: Done. Thank you very much for the review.

**Comment by Rechi** [comment]
@AdityaGupta030697 please squash your commits and get rid of the merge commit. We don't want that in our git history.

**Pull request closed without merging**